src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -134,7 +167,57 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 		if (this._model.secondaryElement?.properties) {
 			secondaryProps = this._model.secondaryElement.properties;
 		}
-		this.populateTable(columns, this.convertPropertiesToTableRows(primaryProps, secondaryProps));
+
+		const tableRows = this.convertPropertiesToTableRows(primaryProps, secondaryProps);
+		this.setSummaryElement(this.getExpensivePropertySummary(tableRows));
+		this.populateTable(columns, tableRows);
+	}
+
+	private getExpensivePropertySummary(tableRows: { [key: string]: string }[]): string[] {
```

Comment:
This could use a description. What is this summary for and what format will it be output in?